Formula/tsduck.rb

```diff
@@ -1,8 +1,8 @@
 class Tsduck < Formula
   desc "MPEG Transport Stream Toolkit"
   homepage "https://tsduck.io/"
-  url "https://github.com/tsduck/tsduck/archive/v3.34-3197.tar.gz"
-  sha256 "5e58f220063a5284080a612cc0f62e7f99329a1b2b1d5763f4b014b2578c95c2"
+  url "https://github.com/tsduck/tsduck/archive/v3.35-3258.tar.gz"
+  sha256 "9fc1679363e24423329d7f24b0dedd2aa877c98776f5063b7b15dc6b891ba5f6"
```

Comment:
```suggestion
  sha256 "7c28d04fad9ef8415dc02d56f377e5b2b4c3f173dcc9487a906ee73da38f4e41"
```

Reply:
Thanks @chenrui333. This is strange, the command `brew bump-formula-pr` is run as part of a script which gets the URL and sha256 of the latest version. If I rerun the same script again, I now get the same sha256 as you but the same URL as before. As if the output of `curl -sL (URL)` had changed.

Script: https://github.com/tsduck/tsduck/blob/master/.homebrew/brew-bump-formula-pr.sh